qutebrowser/config/configdata.py

```diff
@@ -389,7 +389,8 @@ def data(readonly=False):
              "* `{id}`: The internal window ID of this window.\n"
              "* `{scroll_pos}`: The page scroll position.\n"
              "* `{host}`: The host of the current web page.\n"
-             "* `{backend}`: Either 'webkit' or 'webengine'"),
+             "* `{backend}`: Either 'webkit' or 'webengine'"
```

Comment:
Dang, that's what I get for copying/pasting the 'private line' I had inserted arbitrarily in the middle of that block to the end. Sorry!